src/app/interfaces/title/title.interface.ts

```diff
@@ -0,0 +1,12 @@
+export interface InnerItem {
+  id: number;
+  title: string;
+  answer?: string;
+  item?: Object[];
+}
+
+export interface Title {
```

Comment:
`Title` it is very common name, that doesn't describe entity at all. Please change to more appropriate name e.g. `FaqItem`

Also all your props are optional, is this correct? Do we really need the id prop? The users when will create them own structure should follow the your interface.

Reply:
Id property is not necessary.
Why we can get rid of InnerItem?

Reply:
Closed.

Reply:
was applied